src/components/Book.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import { removeBook } from '../redux/books/books';
+
+const Book = (props) => {
+  const { book } = props;
+
+  const {
+    id, genre, title, author,
+  } = book;
+
+  const dispatch = useDispatch();
+
+  const removesBook = () => {
+    dispatch(
+      removeBook({
+        id,
+      }),
+    );
+  };
+
+  return (
+    <div className="Book-Container">
+      <div className="Book-Info">
+        <p className="Genre">{genre}</p>
+        <h3 className="Title">{title}</h3>
+        <p className="Author">{author}</p>
+        <div className="Book-Edits">
+          <button type="button" className="Edit-Item">Comments</button>
+          <span className="Edit-Item">|</span>
+          <button type="button" className="Edit-Item" onClick={removesBook}>Remove</button>
+          <span className="Edit-Item">|</span>
+          <button type="button" className="Edit-Item">Edit</button>
+        </div>
```

Comment:
**Changes Required**
- The typography looks good. However, the colors don't match the design of Zeplin. Let us try to get as close to the design as possible.